cLIMS/organization/views.py

```diff
@@ -1616,10 +1616,15 @@ class DcicView(View):
     def get(self,request,prj_pk):
         projectId = prj_pk
         context = {}
+        sets={}
         project = Project.objects.get(pk=projectId)
-        experiments=Experiment.objects.filter(project=projectId).order_by('-pk')
+        #experiments=Experiment.objects.filter(project=projectId).order_by('-pk')
+        expSet = ExperimentSet.objects.filter(project=projectId).order_by('-pk')
```

Comment:
@Snehal1894 what's up here ? something with `experiementset`-s ?

which problem is it related to - DCIC or GEO ?
is this related to the new feature that you were describing to Job ? - like DCIC exporting experiment sets or something like that ?

Reply:
Yes. This is related to DCIC update. I needed the ExperimentSet Objects and also exps in it to send to html page.
so here dictionary is created  sets.
It has {Expset object: [all exp objects]}

Reply:
@Snehal1894 could you remove the commented code then ?

old code will remain in the history anyways

Reply:
Yes yes, we can remove the commented lines.